fe/fe-core/src/main/antlr4/org/apache/doris/nereids/DorisParser.g4

```diff
@@ -82,6 +88,22 @@ planType
     | ALL // default type
     ;
 
+withRemoteStorageSystem
+    : WITH S3 LEFT_PAREN
+        brokerProperties+=remoteStorageProperty (COMMA brokerProperties+=remoteStorageProperty)*
+        RIGHT_PAREN
+    | WITH HDFS LEFT_PAREN
+        brokerProperties+=remoteStorageProperty (COMMA brokerProperties+=remoteStorageProperty)*
```

Comment:
2. it is same as `PROPERTIES LEFT_PAREN fileProperties+=tvfProperty (COMMA fileProperties+=tvfProperty)* RIGHT_PAREN)`, extract to new clause